packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -865,69 +835,51 @@ export const audiusBackend = ({
   }
 
   /**
-   * Retrieves the user's eth associated wallets from IPFS using the user's metadata CID and creator node endpoints
+   * Retrieves both the user's ETH and SOL associated wallets from the user's metadata CID
    * @param user The user metadata which contains the CID for the metadata multihash
    * @returns Object The associated wallets mapping of address to nested signature
    */
-  // TODO(C-2719)
-  async function fetchUserAssociatedEthWallets(user: User) {
-    const cid = user?.metadata_multihash ?? null
-    if (cid) {
-      const metadata = await fetchCID(cid, /* asUrl */ false)
-      if (metadata?.associated_wallets) {
-        return metadata.associated_wallets
-      }
-    }
-    return null
-  }
+  async function fetchUserAssociatedWallets({
+    user,
+    sdk
+  }: {
+    user: UserMetadata
+    sdk: AudiusSdk
+  }) {
+    if (!user?.metadata_multihash) return null
 
-  /**
-   * Retrieves the user's solana associated wallets from IPFS using the user's metadata CID and creator node endpoints
-   * @param user The user metadata which contains the CID for the metadata multihash
-   * @returns Object The associated wallets mapping of address to nested signature
-   */
-  // TODO(C-2719)
-  async function fetchUserAssociatedSolWallets(user: User) {
-    const cid = user?.metadata_multihash ?? null
-    if (cid) {
-      const metadata = await fetchCID(cid, /* asUrl */ false)
-      if (metadata?.associated_sol_wallets) {
-        return metadata.associated_sol_wallets
-      }
-    }
-    return null
-  }
+    const { data } = await sdk.full.cidData.getMetdata({
+      metadataId: user?.metadata_multihash
+    })
 
-  /**
-   * Retrieves both the user's ETH and SOL associated wallets from the user's metadata CID
-   * @param user The user metadata which contains the CID for the metadata multihash
-   * @returns Object The associated wallets mapping of address to nested signature
-   */
-  // TODO(C-2719)
-  async function fetchUserAssociatedWallets(user: UserMetadata) {
-    const cid = user?.metadata_multihash ?? null
-    if (cid) {
-      const metadata = await fetchCID(cid, /* asUrl */ false)
-      return {
-        associated_sol_wallets: metadata?.associated_sol_wallets ?? null,
-        associated_wallets: metadata?.associated_wallets ?? null
-      }
+    if (!data?.data) return null
+
+    return {
+      associated_sol_wallets: data.data.associatedSolWallets ?? null,
```

Comment:
lol `data.data` weird but ok

Reply:
yeah it's because of how this endpoint is written. didnt' want to refactor that in _this_ PR